package.json

```diff
@@ -1,99 +1,110 @@
-					"when": "config.ConEmu.showTitlebarIcon"
-				}
-			]
-		}
-	},
-	"scripts": {
-		"vscode:prepublish": "tsc -p ./",
-		"compile": "tsc -watch -p ./",
-		"postinstall": "node ./node_modules/vscode/bin/install",
-		"test": "node ./node_modules/vscode/bin/test"
-	},
-	"devDependencies": {
-		"@types/node": "^7.0.5",
-		"typescript": "^2.1.5",
-		"vscode": "^1.0.3"
-	}
-}
\ No newline at end of file
+{
+	"name": "vscode-conemu",
+	"displayName": "ConEmu Launcher",
+	"description": "ConEmu Launcher for VS Code",
+	"version": "1.0.6",
+	"author": "Josh Byrnes",
+	"publisher": "j-byrnes",
+	"engines": {
+		"vscode": "^1.5.0"
+	},
+	"categories": [
+		"Other"
+	],
+	"icon": "images/conemu-icon.png",
+	"galleryBanner": {
+		"color": "#999",
+		"theme": "light"
+	},
+	"activationEvents": [
+		"onCommand:vscode.conemu"
+	],
+	"repository": {
+		"type": "git",
+		"url": "https://github.com/ipatalas/vscode-conemu"
+	},
+	"main": "./out/src/extension",
+	"contributes": {
+		"commands": [
+			{
+				"command": "vscode.conemu",
+				"title": "Open in ConEmu",
+				"icon": {
+					"dark": "./images/dark-icon.png",
+					"light": "./images/light-icon.png"
+				}
+			}
+		],
+		"configuration": {
+			"title": "ConEmu Launcher",
+			"properties": {
+				"ConEmu.path": {
+					"type": "string",
+					"description": "Path to ConEmu executable (ConEmu.exe/ConEmu64.exe)"
+				},
+				"ConEmu.reuseInstance": {
+					"type": "boolean",
+					"description": "Flag indicating whether to use existing instance of ConEmu or open new window",
+					"default": true
+				},
+				"ConEmu.showTitlebarIcon": {
+					"type": "boolean",
+					"description": "Flag indicating whether to show button on titlebar",
+					"default": false
+				},
+				"ConEmu.showMessageInOutputPanel": {
+					"type": "boolean",
+					"description": "Determines whether to show messages in output panel",
+					"default": false
+				},
+				"ConEmu.customRunOption": {
+					"type": "string",
+					"description": "The ConEmu run option to use (i.e. {Shells::PowerShell (Admin)} / @taskfile.txt)"
+				}
+			}
+		},
+		"keybindings": [
+			{
+				"command": "vscode.conemu",
+				"key": "shift+alt+c",
+				"when": "!terminalFocus"
+			}
+		],
+		"menus": {
+			"explorer/context": [
+				{
+					"command": "vscode.conemu",
+					"group": "navigation@99"
+				}
+			],
+			"editor/title/context": [
+				{
+					"command": "vscode.conemu"
+				}
+			],
+			"editor/title": [
+				{
+					"command": "vscode.conemu",
+					"group": "navigation",
+					"when": "config.ConEmu.showTitlebarIcon"
+				}
+			]
+		}
+	},
+	"scripts": {
+		"vscode:prepublish": "tsc -p ./",
+		"compile": "tsc -watch -p ./",
+		"postinstall": "node ./node_modules/vscode/bin/install",
+		"test": "node ./node_modules/vscode/bin/test"
+	},
+	"devDependencies": {
+		"@types/node": "^7.0.5",
+		"typescript": "^2.1.5",
+		"vscode": "^1.1.30"
+	},
+	"dependencies": {
+		"child": "0.0.3",
+		"fs": "0.0.1-security",
+		"path": "^0.12.7"

```

Comment:
Why do you need these dependencies? There are internal node.js functions that are already used and I don't see a point of adding a dependency for that. They look malicious anyway. I'm not gonna accept those changes unless you can point an advantage of each of them over internal functions.

Reply:
I apologize, for making the change as this is the first pull request I ever made and not aware of protocol. Do I change it back and resubmit?

Reply:
Actually, I'm not sure how they were added either.  I removed them

Reply:
One question on these dependencies... is there a way for me to trace them as I do not recall ever adding them myself... would there be a reason why these would have been added, is there a way for me to trace back to their source on why they were added?

Reply:
Well, it's usually a developer's action and I don't know if there is a way to trace it. Since it looks malicious maybe you have also some malicious software installed which scans your projects and tries to inject stuff but that's a long shot.

Reply:
I'm going to see if I can research this further and if I find anything, I will let you know.  I did confirm that the packages.json file I updated doesn't include these references any more.